src/api/app/controllers/build_controller.rb

```diff
@@ -12,7 +12,7 @@ def index
       return
     end
 
-    if User.current.is_admin?
+    if User.possibly_nobody.is_admin?
```

Comment:
Is that possible that an user nobody is admin? Wouldn't a  ```User.current && User.current.is_admin?``` express better the business logic?

Reply:
Not if User.current returns nobody - which is what I'm trying to make explicit with this PR. *Nothing* else. Refactor such code later to make sense: sure, but what you say is out of scope.